desc/compute/_qs.py

```diff
@@ -94,7 +99,7 @@ def compute_boozer_coords(
 
     # covariant Boozer components: I = B_theta, G = B_zeta (in Boozer coordinates)
     idx0 = B_transform.basis.get_idx(M=0, N=0)
-    B_theta_mn = B_transform.fit(data["B_theta"])
+    B_theta_mn = B_transform.fit(data["B_theta"]) * orientation
```

Comment:
I'm still skeptical of this and the similar correction for `I`. I would like to have some proof for it before we commit to it